BSB_lan.ino

```diff
@@ -8076,7 +8078,8 @@ boolean mqtt_connect()
     MQTTPass = MQTTPassword;
   if(MQTTClient == NULL)
   {
-    MQTTClient = new PubSubClient(mqtt_client);  //Luposoft: change instance
+    mqtt_client = new EthernetClient();
+    MQTTClient = new PubSubClient(mqtt_client[0]);
```

Comment:
It is working? Fine if yes.

Reply:
oh, yes it works! Thanks a lot

Reply:
At the top you have created `mqtt_client` as either an `EthernetClient` or a `WiFiSpiClient`instance. So you would have to do the same here when creating a new instance with `#ifdef WIFI`.

Reply:
```
#ifdef WIFI
  max_cul = new WiFiSpiClient();
#else
  max_cul = new EthernetClient();
#endif
```